fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/impl/object/builder/FabricEntityType.java

```diff
@@ -25,8 +25,8 @@
 	private final int maxTrackDistance, trackTickInterval;
 	private final Boolean alwaysUpdateVelocity;
 
-	public FabricEntityType(EntityType.EntityFactory<T> factory, EntityCategory category, boolean bl, boolean summonable, boolean fireImmune, boolean spawnableFarFromPlayer, int i, int j, EntityDimensions entityDimensions, int maxTrackDistance, int trackTickInterval, Boolean alwaysUpdateVelocity) {
-		super(factory, category, bl, summonable, fireImmune, spawnableFarFromPlayer, i, j, entityDimensions);
+	public FabricEntityType(EntityType.EntityFactory<T> factory, EntityCategory category, boolean bl, boolean summonable, boolean fireImmune, boolean spawnableFarFromPlayer, int maxSpawnDistance, int minSpawnDistance, EntityDimensions entityDimensions, int maxTrackDistance, int trackTickInterval, Boolean alwaysUpdateVelocity) {
+		super(factory, category, bl, summonable, fireImmune, spawnableFarFromPlayer, maxSpawnDistance, minSpawnDistance, entityDimensions);
```

Comment:
Is this spawn or despawn distance? should be `despawnStartDistance` and `immediateDespawnDistance` imo

Reply:
Yeah, I like those names much more.

Reply:
Actually, on second thought, I'm not sure about those names.
My reasoning is basically if you see `maxDespawnDistance(int)` you instantly know what the code does, but `despawnStartDistance` could be the point that mobs start despawning from the player or the point that they start despawning from far away.
Keep in mind, there isn't a method to modify `minSpawnDistance`.

Reply:
or at least update them to `despawn` than `spawn` distance